src/card-not-found.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Section from 'elements/card-section';
+import Card from 'elements/card';
+
+export default ({ message }) => {
+  return (
+    <Card title="Not Found" enableLeftHeader={false}>
+      <Section message={message} isCentered={true} />
+    </Card>
+  );
+};
```

Comment:
We did not have a default way of showing this other than some text in MainContainer, so I pulled it out into it's own separate component.